internal/api/#stats.go#

```diff
@@ -0,0 +1,191 @@
+package api
```

Comment:
@khrm 
Is this file intended in the commit?

Reply:
No. Removed it.